docs/source/en/training/text_inversion.mdx

```diff
@@ -14,74 +14,83 @@ specific language governing permissions and limitations under the License.
+```bash
+accelerate config default
+```
 
-## Example: Running locally 
+Or if your environment doesn't support an interactive shell like a notebook, you can use:
 
-The `textual_inversion.py` script [here](https://github.com/huggingface/diffusers/blob/main/examples/textual_inversion) shows how to implement the training procedure and adapt it for stable diffusion.
+```bash
+from accelerate.utils import write_basic_config
 
-### Installing the dependencies
+write_basic_config()
+```
 
-Before running the scripts, make sure to install the library's training dependencies.
+Finally, you try and [install xFormers](https://huggingface.co/docs/diffusers/main/en/training/optimization/xformers) to reduce your memory footprint with xFormers memory-efficient attention. Once you have xFormers installed, add the `--enable_xformers_memory_efficient_attention` argument to the training script. xFormers is not supported for Flax.
 
-```bash
-pip install diffusers[training] accelerate transformers
-```
+## Upload model to Hub
 
-And initialize an [🤗Accelerate](https://github.com/huggingface/accelerate/) environment with:
+If you want to store your model on the Hub, add the following argument to the training script:
 
 ```bash
-accelerate config
+--push_to_hub
 ```
 
+## Save and load checkpoints
 
-### Cat toy example
-
-You need to accept the model license before downloading or using the weights. In this example we'll use model version `v1-4`, so you'll need to visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree.
+It is often a good idea to regularly save checkpoints of your model during training. This way, you can resume training from a saved checkpoint if your training is interrupted for any reason. To save a checkpoint, pass the following argument to the training script to save the full training state in a subfolder in `output_dir` every 500 steps:
 
-You have to be a registered user in 🤗 Hugging Face Hub, and you'll also need to use an access token for the code to work. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).
+```bash
+--checkpointing_steps=500
+```
 
-Run the following command to authenticate your token
+To resume training from a saved checkpoint, pass the following argument to the training script and the specific checkpoint you'd like to resume from:
 
 ```bash
-huggingface-cli login
+--resume_from_checkpoint="checkpoint-1500"
 ```
 
-If you have already cloned the repo, then you won't need to go through these steps.
+## Finetuning
```

Comment:
We can add a section here about intermediate logging - basically, during the training process, we periodically save generated images, so that you have a better idea of how the training progress 

You can find relevant arguments in this section of the training scripts:
https://github.com/huggingface/diffusers/blob/main/examples/textual_inversion/textual_inversion.py#L307

`--validation_prompt`: the prompt used to generate samples, by default is `None` and we will not do intermediate logging, intermediate logging is activated when the user explicitly passes a string value for `validation_prompt`
`--num_validation_images`: number of samples to generate 
`--validation_steps`: do the logging every x training steps